Autocoders/Python/src/fprime_ac/generators/visitors/SerialHVisitor.py

```diff
@@ -187,18 +186,28 @@ def _get_enum_string_list(self, enum_list):
         enum_list = enum_list[1]
         enum_str_list = []
         for e in enum_list:
-            # No value, No comment
-            if (e[1] is None) and (e[2] is None):
-                s = "%s," % (e[0])
-            # No value, With comment
-            elif (e[1] is None) and (e[2] is not None):
-                s = "{},  // {}".format(e[0], e[2])
-            # With value, No comment
-            elif (e[1] is not None) and (e[2] is None):
-                s = "{} = {},".format(e[0], e[1])
-            # With value and comment
-            elif (e[1] is not None) and (e[2] is not None):
-                s = "%s = %s,  // %s" % (e)
+            # Without value
```

Comment:
There are too many nested statements in my opinion.

Wouldn't something like below be better?

```py
    def _get_enum_string_list(self, enum_list):
        """"""
        enum_tuple = enum_list[0]
        enum_list = enum_list[1]
        enum_str_list = []
        for e in enum_list:
            # No value, No comment
            if (e[1] is None) and (e[2] is None):
                s = f"{e[0]},"
            elif e[1] is None:
                s = f"{e[0]},  // {e[2]}"
            elif e[2] is None:
                s = f"{e[0]} = {e[1]},"
            else:
                s = f"{e} = {e},  // {e}"
            enum_str_list.append(s)

        return (enum_tuple, enum_str_list)
```